tests/test_diagnosis_routes.py

```
@@ -0,0 +1,142 @@
+import pytest
+from httpx import AsyncClient, ASGITransport
+from unittest.mock import AsyncMock, MagicMock
+from main import app
+from api.frame.frame_routes import uid_queues
+from api.frame.TimedQueue import TimedQueue
+import numpy as np
+from PIL import Image
+
+
+@pytest.mark.asyncio
+async def test_get_diagnosis_success():
```

Ruff (0.11.9)

Missing return type annotation for public function `test_get_diagnosis_success`

Add return type annotation: `None`

(ANN201)